frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/ticketGroups/groups/groupActionMenu/groupSettingsForm/groupSettingsForm.component.tsx

```diff
@@ -329,7 +326,7 @@ export const GroupSettingsForm = ({ value, onSubmit, onCancel }: GroupSettingsFo
 								size="medium"
 								fullWidth={false}
 								onClick={handleSubmit(onClickSubmit)}
-								disabled={!isValid || isEmpty(dirtyFields)}
+								disabled={!getFormIsValid()}
```

Comment:
The form validates if you change a field and then change it back to the original value. Should this be happening?
This is most noticeable when you open the colorPicker and immediately close it. The form will validate as if you have set a new value.